app/views/decidim/participatory_documents/admin/documents/index.html.erb

```diff
@@ -1,24 +1,51 @@
 <div class="card full-height">
   <div class="card-divider">
-    <h2 class="card-title">
+    <h2 class="card-title flex--sbc">
       <%= t(".title") %>
+      <div class="flex--cc flex-gap--1">
+        <% if allowed_to? :create, :participatory_document %>
```

Comment:
In here, we want the action to create a new document if there's none yet.
If not, we want the action to "edit the current document"